bind.go

```diff
@@ -0,0 +1,83 @@
+package fiber
+
+import (
+	"errors"
+
+	"github.com/gofiber/fiber/v3/binder"
+	"github.com/gofiber/fiber/v3/utils"
+)
+
+type CustomBinder interface {
+	Name() string
+	Parse(Ctx, any) error
+}
+
+type Bind struct {
+	ctx    Ctx
+	should bool
+}
+
+func (b *Bind) Should() {
+	b.should = true
+}
+
+func (b *Bind) Must() {
+	b.should = false
+}
+
+func (b *Bind) Custom(name string, dest any) error {
+	binders := b.ctx.App().customBinders
+
+	for _, binder := range binders {
+		if binder.Name() == name {
+			return binder.Parse(b.ctx, dest)
+		}
+	}
+
+	return errors.New("fiber: custom binder not found, please be sure to enter right name!")
+}
+
+func (b *Bind) Header(out any) error {
+	return binder.HeaderBinder.Bind(b.ctx.Request(), out)
```

Comment:
shouldn't the Bind.Query also check the customBinder ? otherwise the functionalities are not changeable with custom binders

Reply:
I think we don't need it at the moment. We can take a look it next pr